.github/workflows/node.yaml

```diff
@@ -0,0 +1,28 @@
+name: Node.js CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x, 15.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: npm install
+    - run: npm start
+    - run: npm run docs
+    - run: npm run build --if-present
```

Comment:
Will this add the compiled files to the current PR? Else, what's the end result?

Reply:
i am not well in github actions i done this on reading github node workflow doc 
here is link of two docs [Github node workflow](https://docs.github.com/en/actions/guides/building-and-testing-nodejs) 
i will delete the linter and css workflow we can add the eslinter setup

Reply:
Any updates on this?

Reply:
> > Any updates on this?
> 
> can you please elaborate the conflict and i will delete this pull and make a new pull with all update

@eltonmesquita can you please elaborate on it?